### DISCUSSION TITLE
Postgres not compatible

### DISCUSSION THREAD

**Discussion by Deezer87** [TandoorRecipes/recipes#2004] [Q&A]
My Tandoor container crashed. When I restarted it from my Portainer Stack, I am receiving this error on the DB container:
The data directory was initialized by PostgreSQL version 11, which is not compatible with this version 14.2.

Any way to fix this? Its not a big issue for me to restart from scratch as I only have a handful of recipes in there but I would have to track them down again. And I would like to figure out what happened.

**Comment by vabene1111** [comment]
Sorry for the late reply.

You must have changed pg versions. You can revert the container to the old version, dump all data and load it into the new version container.

This is generally the only way as far as I know to upgrade pg versions
